### ISSUE TITLE
Redesign the detailed view page

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#63]
## User story

1. As a facility manager
2. I need SDs to redesign the detailed view 
3. So that it is more clear and less confusing

## Acceptance criteria

- [ ] The facility manager can identify that the detailed view is a new page.
- [ ] The detailed view page clearly states the facility where details are listed and the listed details for that facility

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release
